controllers/permits/special-uses/validate.js

```diff
@@ -36,9 +36,136 @@ const phoneNumber = schema.phoneNumber;
 
 //*******************************************************************
 
+function zipFormat(input){
+	
+	let valid = true;
+
+	if (typeof input === 'number'){
+
+		const inputStr = input + '';
+
+		if (!inputStr.match(/^[0-9]{5}$|^[0-9]{9}$/)){
+
+			valid = false;
+
+		}
+
+	}
+	return valid;
```

Comment:
@brianfunk I had it pass if input wasn't a number so that it would not cause an error for both type, and formatting. Seemed a bit redundant.